src/test/java/eu/europeana/metis/sandbox/controller/DatasetControllerTest.java

```diff
@@ -483,64 +482,134 @@ void processDatasetFromOAI_differentXsltFileType_expectFail() throws Exception {
   }
 
   @Test
-  void retrieveDataset_expectSuccess() throws Exception {
+  void retrieveDatasetProgress_expectSuccess() throws Exception {
     var message1 = "cvc-complex-type.4: Attribute 'resource' must appear on element 'edm:object'.";
     var message2 = "cvc-complex-type.2.4.b: The content of element 'edm:ProvidedCHO' is not complete.";
     var error1 = new ErrorInfoDto(message1, Status.FAIL, List.of("1", "2"));
     var error2 = new ErrorInfoDto(message2, Status.FAIL, List.of("3", "4"));
     var errors = List.of(error1, error2);
     var createProgress = new ProgressByStepDto(Step.HARVEST_FILE, 10, 0, 0, List.of());
     var externalProgress = new ProgressByStepDto(Step.VALIDATE_EXTERNAL, 7, 3, 0, errors);
-    var datasetInfoDto = new DatasetInfoDto("12345", "Test", LocalDateTime.MIN, Language.NL,
-        Country.NETHERLANDS, false, false);
     var tiersZeroInfo = new TiersZeroInfo(new TierStatistics(0, Collections.emptyList()),
         new TierStatistics(0, Collections.emptyList()));
     var report = new ProgressInfoDto("https://metis-sandbox",
-        10L, 10L, List.of(createProgress, externalProgress), datasetInfoDto, "", emptyList(),
+        10L, 10L, List.of(createProgress, externalProgress), false, "", emptyList(),
         tiersZeroInfo);
     when(datasetReportService.getReport("1")).thenReturn(report);
 
-    mvc.perform(get("/dataset/{id}", "1"))
+    mvc.perform(get("/dataset/{id}/progress", "1"))
         .andExpect(status().isOk())
         .andExpect(jsonPath("$.status",
             is("COMPLETED")))
         .andExpect(jsonPath("$.progress-by-step[1].errors[0].message",
-            is(message1)))
-        .andExpect(jsonPath("$.dataset-info.dataset-id", is("12345")))
-        .andExpect(jsonPath("$.dataset-info.dataset-name", is("Test")))
-        .andExpect(jsonPath("$.dataset-info.creation-date", is("-999999999-01-01T00:00:00")))
-        .andExpect(jsonPath("$.dataset-info.language", is("Dutch")))
-        .andExpect(jsonPath("$.dataset-info.country", is("Netherlands")))
-        .andExpect(jsonPath("$.dataset-info.record-limit-exceeded", is(false)))
-        .andExpect(jsonPath("$.dataset-info.transformed-to-edm-external", is(false)));
+            is(message1)));
```

Comment:
my suggestion is to add this missing check as well.
```
 .andExpect(jsonPath("$.progress-by-step[1].errors[1].message",
           is(message2)));
```

Reply:
Done